<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gz patcher</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabwidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_rom">
       <attribute name="title">
        <string>Patch a ROM</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QGroupBox" name="groupbox_rom">
          <property name="title">
           <string>ROM</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="label_rom">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No ROM selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_rom">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_ucode">
          <property name="title">
           <string>Line microcode</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="checkbox_ucode">
             <property name="text">
              <string>Inject line microcode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_ucode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No microcode ROM selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_ucode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_wad">
       <attribute name="title">
        <string>Patch a WAD</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QGroupBox" name="groupbox_wad">
          <property name="title">
           <string>WAD</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="label_wad">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No WAD selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_wad">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_extrom">
          <property name="title">
           <string>ROM</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="checkbox_extrom">
             <property name="text">
              <string>Use external ROM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_extrom">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No ROM selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_extrom">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_options">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_remap">
             <property name="text">
              <string>Controller Remapping</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="combobox_remap">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Raphnet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_id">
             <property name="text">
              <string>Channel ID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="combobox_id">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom...</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_title">
             <property name="text">
              <string>Channel Title</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="combobox_title">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom...</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_region">
             <property name="text">
              <string>Region</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="combobox_region">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <item>
              <property name="text">
               <string>Japan</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>USA</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Europe</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Region Free</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_iso">
       <attribute name="title">
        <string>Patch an ISO</string>
       </attribute>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QGroupBox" name="groupbox_iso">
          <property name="title">
           <string>ISO</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QLabel" name="label_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No ISO selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_extrom_iso">
          <property name="title">
           <string>ROM</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="checkbox_extrom_iso">
             <property name="text">
              <string>Use external ROM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_extrom_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No ROM selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_extrom_iso">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_extrom_mq_iso">
          <property name="title">
           <string>MQ ROM</string>
          </property>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QCheckBox" name="checkbox_extrom_mq_iso">
             <property name="text">
              <string>Use external ROM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_extrom_mq_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>No ROM selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_extrom_mq_iso">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupbox_options_iso">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label_title_iso">
             <property name="text">
              <string>Game Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_id_iso">
             <property name="text">
              <string>Game ID</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="combobox_title_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom...</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="combobox_id_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom...</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="combobox_remap_iso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Default</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Raphnet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_remap_iso">
             <property name="text">
              <string>Controller Remapping</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="checkbox_iso_no_trim">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Remove Large Files</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="button_go">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
